main_seed_full_path_test.go

```
@@ -0,0 +1,56 @@
+
+	if err := seedDatabase(context.Background()); err == nil {
+		t.Error("expected error when ent client is nil")
+	}
+}
+
+func TestSeedDatabase_FullSeedPath(t *testing.T) {
+	freshClient, err := ent.Open("sqlite3", "file:ent_seed_full?mode=memory&cache=shared&_fk=1")
+	if err != nil {
+		t.Fatalf("failed to open fresh sqlite client: %v", err)
+	}
+	defer freshClient.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `freshClient.Close` is not checked

(errcheck)